susan/db/rdbms/dhcp.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2017 <thenakliman@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from susan.db import dhcp as dhcp_db
+from susan.db.rdbms.models import dhcp as d_model
+from susan.db import rdbms
+
+
+class Subnet(dhcp_db.DHCPdb):
+    def __init__(self):
+        super(Subnet, self).__init__()
+
+    def create_subnet(self, network, cidr, gateway=None):
```

Comment:
I don't want to make session visibility outside this file because we might have kv. So can we optimize sessions for multiple queries?